src/pages/Login/index.tsx

```diff
@@ -1,7 +1,57 @@
+import { useEffect, useState } from 'react';
+import { SubmitHandler } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { Link, useNavigate } from 'react-router-dom';
+import clsx from 'clsx';
+
+import { login } from 'state/actions/userActions';
+import LoginForm from './components/LoginForm';
+import Spinner from 'components/Spinner';
+import { useAppDispatch } from 'hooks/redux';
+import useStatus from 'hooks/useStatus';
+import routesPaths from 'constants/routesPaths';
+import { FULFILLED, PENDING } from 'constants/actionStatus';
+import { LoginFields } from 'types/forms';
+import { smiles } from 'assets/images'
+import styles from './styles.module.scss';
```

Comment:
I was referring to this
```suggestion
import { login } from 'state/actions/userActions';
import Spinner from 'components/Spinner';
import { useAppDispatch } from 'hooks/redux';
import useStatus from 'hooks/useStatus';
import routesPaths from 'constants/routesPaths';
import { FULFILLED, PENDING } from 'constants/actionStatus';
import { LoginFields } from 'types/forms';
import { smiles } from 'assets/images'

import LoginForm from './components/LoginForm';
import styles from './styles.module.scss';
```

Reply:
a ok, I got it. 

The form I order those import is away to identify what import are related to external libraries/packages and what are related to local files.

to make these changes I would have to make a big refactor in the entire app. Not only in this file so I think is not viably